### PR TITLE
AdoptAWidget: LongPressDraggable

### LINKED ISSUE #69473: Adopt a Widget - LongPressDraggable
The [LongPressDraggable](https://master-api.flutter.dev/flutter/widgets/LongPressDraggable-class.html) widget needs:

- More references

For details about how to participate, see [flutter.dev/adoptawidget](https://flutter.dev/adoptawidget)

### PR DESCRIPTION
**Pull request by kevinrodriguez-io**
AdoptAWidget: LongPressDraggable

This change adds missing "See also" links to the LongPressDraggable widget.

This pull request is:

- [ ] A code sample
- [x] More references
- [ ] More explanation

closes #69473

### CODE REVIEW

**Comment by kevinrodriguez-io** [comment]
@googlebot I signed it!

File: packages/flutter/lib/src/widgets/drag_target.dart [hunk 1]
```diff
@@ -358,6 +358,11 @@ class Draggable<T extends Object> extends StatefulWidget {
 }
 
 /// Makes its child draggable starting from long press.
+///
+/// See also:
+///
+///  * [Draggable]
```
Comment (goderbauer) [inline packages/flutter/lib/src/widgets/drag_target.dart:364]: there should be an explanation here telling me why I should also look at "draggable".

File: packages/flutter/lib/src/widgets/drag_target.dart [hunk 2]
```diff
@@ -358,6 +358,11 @@ class Draggable<T extends Object> extends StatefulWidget {
...
+///  * [DragTarget], a widget that receives data when the [LongPressDraggable] is dragged.
```
Comment (goderbauer) [inline packages/flutter/lib/src/widgets/drag_target.dart:365]: Doesn't it only receive the data when the draggable is dropped on the target?

File: packages/flutter/lib/src/widgets/drag_target.dart (hunk 1 above)
Comment (kevinrodriguez-io) [reply packages/flutter/lib/src/widgets/drag_target.dart:364]: Good catch, I will update both

File: packages/flutter/lib/src/widgets/drag_target.dart (hunk 2 above)
Comment (kevinrodriguez-io) [reply packages/flutter/lib/src/widgets/drag_target.dart]: @goderbauer kinda; It provides the following events:

this.onWillAccept -> Receives data
this.onAccept -> Receives data
this.onAcceptWithDetails -> Receives data
this.onLeave -> Doesn't
this.onMove -> Doesn't

But this chain of events are fired on drag.

File: packages/flutter/lib/src/widgets/drag_target.dart (hunk 1 above)
Comment (kevinrodriguez-io) [reply packages/flutter/lib/src/widgets/drag_target.dart]: Done.

**Comment by kevinrodriguez-io** [comment]
@johnpryan Do we Include reference to videos on the docs? [Video](https://www.youtube.com/watch?v=QzA4c4QHZCY)

**Comment by johnpryan** [comment]
@kevinrodriguez-io yes, you can add that using this syntax:

```
/// {@youtube 560 315 <youtube_video_url>}
```

File: packages/flutter/lib/src/widgets/drag_target.dart [hunk 3]
```diff
@@ -358,6 +358,11 @@ class Draggable<T extends Object> extends StatefulWidget {
 }
 
 /// Makes its child draggable starting from long press.
+///
+/// See also:
+///
+///  * [Draggable], similar to the [LongPressDraggable] widget but happens immediately.
+///  * [DragTarget], a widget that receives drags.
```
Comment (johnpryan) [inline packages/flutter/lib/src/widgets/drag_target.dart:365]:
```suggestion
///  * [DragTarget], a widget that receives data when Draggable widget is dropped.
```

**Comment by johnpryan** [comment]
gentle ping @kevinrodriguez-io just one small comment. did you want to add the YouTube url as well?

**Comment by kevinrodriguez-io** [comment]
Yes! It would be awesome to have it

**Review by johnpryan — APPROVED** [review]

**Review by Piinks — COMMENTED** [review]
Hi @kevinrodriguez-io can you update this with the latest from master? That will resolve the unrelated failing checks here so we can get this merged. :)

**Comment by kevinrodriguez-io** [comment]
@Piinks Sure! In a moment.

**Comment by goderbauer** [comment]
@kevinrodriguez-io Do you still have plans to follow up on @Piinks comment?

**Comment by kevinrodriguez-io** [comment]
> @kevinrodriguez-io Do you still have plans to follow up on @Piinks comment?

I thought I did, weird thing. Let me try again.

**Comment by goderbauer** [comment]
@kevinrodriguez-io Looks like you still need to rebase this to get the tests to pass as suggested by @Piinks above. Any plans to do that soon?

**Comment by kevinrodriguez-io** [comment]
I'm trying right now

**Comment by goderbauer** [comment]
I am going to close this one since there hasn't been any updates. Feel free to reopen when you've the time for it again.
